Home_Work_1/Task_2.py

```diff
@@ -0,0 +1,15 @@
+# Задача 4: Петя, Катя и Сережа делают из бумаги журавликов. Вместе они сделали S журавликов. 
+# Сколько журавликов сделал каждый ребенок, если известно, что Петя и Сережа сделали одинаковое количество журавликов,
+# а Катя сделала в два раза больше журавликов, чем Петя и Сережа вместе?
+# *Пример:*
+# 6 -> 1  4  1
+# 24 -> 4  16  4
+# 60 -> 10  40  10
+
+Sum = int (input('Enter a number: '))
+
+P = (Sum / 3) / 2
+S = (Sum / 3) / 2
```

Comment:
Приоритет у операции деления одинаковый, значит скобки не нужны. Кроме того, чтобы не использовать функцию `int` в 15 строке, можно использовать здесь целочисленное деление.
```suggestion
P = Sum // 3 // 2
S = Sum // 3 // 2
```